src/main/java/com/pnu/dev/shpaltaif/controller/PostController.java

```diff
@@ -1,8 +1,33 @@
 package com.pnu.dev.shpaltaif.controller;
 
+import com.pnu.dev.shpaltaif.domain.Post;
+import com.pnu.dev.shpaltaif.service.PostService;
+import com.pnu.dev.shpaltaif.service.PublicAccountService;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
 
 @Controller
+@RequestMapping("/posts")
 public class PostController {
 
+    private final PostService postService;
+
+    private final PublicAccountService publicAccountService;
```

Comment:
It's used for dropdown list of categories for filtering

Reply:
It's public controller...
Categories will be available for UI anyway, because we have ControllerAdvice that sets them.
And anyway, I don't see the reason to have publicAccountService here, it remains unused.

Reply:
ControllerAdvice sets categories only to requests which URI doesn't start with "/admin"
And the needs of publicAccountService is admin's ability to filter posts by it's author
![image](https://user-images.githubusercontent.com/33934184/95013180-4c734700-0647-11eb-9cbf-3044fc6ef3f4.png)